shell/platform/android/io/flutter/embedding/engine/plugins/broadcastreceiver/BroadcastReceiverAware.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.embedding.engine.plugins.broadcastreceiver;
+
+import android.support.annotation.NonNull;
+
+/**
+ * {@link FlutterPlugin} that wants to know when it is running within a {@link BroadcastReceiver}.
```

Comment:
nit: Having trouble parsing this phrase.

Maybe:

```suggestion
 * A {@link FlutterPlugin} that wants to know when it is running within a {@link BroadcastReceiver}.
```

Reply:
done